tests/unittests/OnnxImporterTest.cpp

```diff
@@ -1636,3 +1636,19 @@ TEST(onnx, batchNormPR2304) {
       llvm::dyn_cast<BatchNormalizationNode>(trNode->getInput().getNode());
   EXPECT_NE(nullptr, bnNode);
 }
+
+/// Test constructor for auto loading inputs case.
+TEST(onnx, autoLoadInputs) {
+  ExecutionEngine EE;
+  auto &mod = EE.getModule();
+  std::string netFilename(GLOW_DATA_PATH
+                          "tests/models/onnxModels/batchNormPR2304.onnxtxt");
+  auto *F = mod.createFunction("main");
+  Tensor inputTensor(ElemKind::FloatTy, {1, 2, 10, 10});
+  llvm::StringRef inputName = "input";
+  ONNXModelLoader onnxLD(netFilename, {}, {}, *F);
+  auto inputs = onnxLD.getInputVarsMapping();
+  EXPECT_EQ(inputs.size(), 1);
+  EXPECT_EQ(inputs["input"]->getName(), inputName);
+  EXPECT_TRUE(inputTensor.getType().isEqual(inputs["input"]->getType()));
```

Comment:
nit: since we already have `inputName`, could just use `inputs[inputName]`. And actually, you probably should get `inputs["input"]` into its own variable before this and assert it's not null, then use the variable here.